web/src/engine/websites/TempleScan.ts

```diff
@@ -0,0 +1,70 @@
+import { Tags } from '../Tags';
+import icon from './TempleScan.webp';
+import { Chapter, DecoratableMangaScraper, Manga, type MangaPlugin, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON } from '../platform/FetchProvider';
+
+type APIManga = {
+    title: string,
+    series_slug: string,
+    Season?: {
+        Chapter: APIChapter[]
+    }[]
+}
+
+type APIChapter = {
+    index: string,
+    chapter_name: string,
+    chapter_title: string | null
+    chapter_slug: string,
+    chapter_data: {
+        images: string[]
+    }
+}
+
+@Common.ImageAjax(true)
+export default class extends DecoratableMangaScraper {
+
+    private readonly apiUrl = 'https://api.templescan.net/api/';
+    private readonly mangaRegexp = new RegExp(`^${this.URI.origin}/(comic/[^/]+)$`);
```

Comment:
May use `RegExpSafe` instead.

Reply:
I used it before and i dont know why, on this particular branch, CI was yelling that RegExpSafe was undefined.
I didnt had this error in IDE, and branch was up to date with master.

Reply:
Told you > 'Error: ReferenceError: RegExpSafe is not defined'